.github/workflows/push-nonroot.yml

```diff
@@ -0,0 +1,30 @@
+name: Build & Push Nonroot Container Image
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: "1"
+      IMAGE: docker.io/${{ github.repository }}
```

Comment:
Can we also push to Quay.io?

Reply:
I guess that should go in a separate PR? Also, with https://github.com/vshn/k8up/issues/359 this becomes mute.